### ISSUE TITLE
Bugs 2/18

### ISSUE THREAD

**Issue by snebtor** [snebtor/Constellations#1]
non-Editmode crashes when receives mouse input. It should not be clickable
save map points to a file and read in restart

**Comment by Izza11** [comment]
non-Editmode crashes when receives mouse input. It should not be clickable - Done
save map points to a file and read in restart - Done

Press 'w' to save map points and 'r' to read them on startup
Press any other key to apply texture
